src/components/POLine/Vendor/VendorForm.js

```diff
@@ -54,4 +65,8 @@ class VendorForm extends Component {
   }
 }
 
+VendorForm.propTypes = {
+  accounts: PropTypes.arrayOf(PropTypes.object).isRequired,
```

Comment:
I think it's better to remove `isRequired`, just handle the case when accounts doesn't come

Reply:
Vendor object has accounts and currencies property by default and their value at least empty array. My solution based on this point. But I could add some default values for the properties